lib/clearbit/base.rb

```diff
@@ -2,5 +2,14 @@ module Clearbit
   class Base < Resource
     endpoint 'https://api.clearbit.com'
     options :format => :json
+
+    def self.version=(value)
+      options headers: {'API-Version' => value}
```

Comment:
this feels a little magical to me. How about `set_options` which augments the original options?